### PR TITLE
rawserial driver for lcdproc

### PR DESCRIPTION
**Pull request by sean-wilkinson**
Hello the rawserial driver has been avalbile since 0.5.7 this will dump raw serial data to the serial port. Hackers/makers can then parse the output with an arduino to display on an LCD. This Pull request adds raw serial as an option in the gui.

### CODE REVIEW

File: sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc.inc [hunk 1]
```diff
@@ -428,6 +428,12 @@ function sync_package_lcdproc() {
 				$config_text .= "Device={$realport}\n";
 				$config_text .= "Size={$lcdproc_config['size']}\n";
 				break;
+			case "rawserial":
+                                $config_text .= "[{$lcdproc_config['driver']}]\n";
+                                $config_text .= "Device={$realport}\n";
+                                $config_text .= "Size={$lcdproc_config['size']}\n";
+                                $config_text .= "UpdateRate=1\n";
+                                break;
```
Comment (rbgarga) [inline sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc.inc]: Just a minor fix required, use TAB to indent. It's the project style

Comment (sean-wilkinson) [reply sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc.inc]: I have changes the spaces to tabs and bumped the portversion in the make file as requested.
Sorry for that; this is my first pull request :)

**Comment by jim-p** [comment]
To effectively track changes and bug/feature requests, we require an entry on the [pfSense Redmine issue tracker](https://redmine.pfsense.org) associated with every pull request, and likewise the Redmine entry should also have a link back to the pull request.

Please associate this PR to a Redmine issue either by locating an existing issue at https://redmine.pfsense.org or by creating a new issue. Add a link to the Redmine issue that points to this PR, and also add a link on the PR to the Redmine issue.

For more information, see https://www.netgate.com/docs/pfsense/development/submitting-a-pull-request-via-github.html

Thanks!

**Comment by rbgarga** [comment]
Superseded by #768

**Pull request closed without merging**